diff -Naur perl-ldap-0.34/data/slapd-conf.in perl-ldap-0.34.oden/data/slapd-conf.in
--- perl-ldap-0.34/data/slapd-conf.in	2007-02-11 00:44:19.000000000 +0100
+++ perl-ldap-0.34.oden/data/slapd-conf.in	2008-03-06 13:08:12.000000000 +0100
@@ -3,13 +3,13 @@
 #
 include		./data/slapd.at.conf
 include		./data/slapd.oc.conf
-schemacheck	off
+#schemacheck	off
 
 #######################################################################
 # ldbm database definitions
 #######################################################################
 
-database	ldbm
+database	bdb
 suffix		"$BASEDN"
 directory	$TESTDB
 rootdn		"$MANAGERDN"
diff -Naur perl-ldap-0.34/data/slapd2-conf.in perl-ldap-0.34.oden/data/slapd2-conf.in
--- perl-ldap-0.34/data/slapd2-conf.in	2007-02-11 00:44:19.000000000 +0100
+++ perl-ldap-0.34.oden/data/slapd2-conf.in	2008-03-06 13:08:20.000000000 +0100
@@ -5,7 +5,7 @@
 include $SCHEMA_DIR/cosine.schema
 include $SCHEMA_DIR/inetorgperson.schema
 include $SCHEMA_DIR/openldap.schema
-schemacheck   	off
+#schemacheck   	off
 pidfile     $TESTDB/slapd.pid
 argsfile    $TESTDB/slapd.args
 
